webservices/resources/totals.py

```diff
@@ -78,6 +85,52 @@ def _resolve_committee_type(self, committee_id=None, committee_type=None, **kwar
         elif committee_type is not None:
             return reports_type_map.get(committee_type)
 
+@doc(
+    tags=['candidate'],
+    description=docs.TOTALS,
+    params={
+        'candidate_id': {'description': docs.CANDIDATE_ID},
+    },
+)
+class CandidateTotalsView(utils.Resource):
+
+    @use_kwargs(args.paging)
+    @use_kwargs(args.totals)
+    @use_kwargs(args.candidate_committee_totals)
+    @use_kwargs(args.make_sort_args(default='-cycle'))
+    @marshal_with(schemas.CommitteeTotalsPageSchema(), apply=False)
+    def get(self, candidate_id, **kwargs):
```

Comment:
I was looking through other code in our resources because I couldn't quite remember how specific URL parameters could come through as `None` (e.g., in `build_query()` below), but what I came across actually had the arguments defined in custom `get()` methods set to `None` by default as well.  Should that be the case here?

As an example, here's a resource a defined in [`webservices/resources/totals.py`](https://github.com/18F/openFEC/blob/develop/webservices/resources/totals.py#L42) that demonstrates what I'm referring to.